src/lib/Form/Factory/FormFactory.php

```diff
@@ -525,8 +525,9 @@ public function deleteRoleAssignment(
         ?string $name = null
     ): FormInterface {
         $name = $name ?: sprintf('delete-role-assignment-%s', md5(implode('/', [
-            $data->getRoleAssignment()->getRole()->id,
-            $data->getRoleAssignment()->getRoleLimitation()->getIdentifier(),
+            $data->getRoleAssignment()->getRole()->id, !empty($data->getRoleAssignment()->getRoleLimitation())
```

Comment:
It's very personal, but I find that complicated to read when going through the code. A ternary method call inside an md5 of an implode of... :)

Reply:
Because you're not so good in reading kung-fu!

Reply:
You know how it goes: maintaining code is twice as hard than initially implementing it. If you're at 100% when writing code, you need to be at 200% when maintaining it.

Conclusion: keep it simple :)